shell/browser/mac/dict_util.mm

```diff
@@ -10,6 +10,10 @@
 #include "base/strings/sys_string_conversions.h"
 #include "base/values.h"
 
+#if !defined(__has_feature) || !__has_feature(objc_arc)
+#error "This file requires ARC support."
+#endif
```

Comment:
Are we gonna need to put this at the top of every file that we convert? If so should it be a macro?

Reply:
@MarshallOfSound we could - i was following Chromium's approach [here](https://chromium.googlesource.com/chromium/src/+/main/docs/mac/arc.md#convention-boilerplate)